metron-platform/metron-integration-test/src/main/java/org/apache/metron/integration/components/KafkaComponent.java

```diff
@@ -65,6 +65,10 @@
 public class KafkaComponent implements InMemoryComponent {
 
   protected static final Logger LOG = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+  public static final long KAFKA_PROPAGATE_TIMEOUT_MS = 10000l;
+  public static final int ZK_SESSION_TIMEOUT_MS = 30000;
+  public static final int ZK_CONNECTION_TIMEOUT_MS = 30000;
+  public static final int KAFKA_ZOOKEEPER_TIMEOUT_MS = 1000000;
```

Comment:
I was on the fence about whether this should just be a string or not, but I think having the numeric constants consistently numeric like this makes it a bit cleaner.